testing/environments/docker/kibana/Dockerfile-snapshot

```diff
@@ -42,3 +39,9 @@ USER kibana
 RUN if [ ${XPACK} = "1" ]; then bin/kibana-plugin install ${DOWNLOAD_URL}/kibana-plugins/x-pack/x-pack-${ELASTIC_VERSION}.zip?c=${CACHE_BUST}; fi
 
 CMD ["/bin/bash", "/usr/local/bin/kibana-docker"]
+
+# Healthcheck create by beats team
+USER root
+RUN yum install -y epel-release && yum clean all
+HEALTHCHECK --interval=1s --retries=300 CMD curl -f http://localhost:5601/api/status | jq '. | .status.overall.state' | grep -q green
```

Comment:
@dliappis  That is how we do the healhcheck for kibana (rather complicated as we need additional packages). Based on https://github.com/elastic/elasticsearch-docker/issues/60#issuecomment-340920511 I assume healthcheck will not show up in our official images. The problem with the above is that we can't do it just in docker-compose as some packages are missing. Are there alternative solutions here?

Reply:
@ruflin Best would be to add all the missing packages to be able to define health checks. For example we include `wget` and `curl` in the elasticsearch image, feel free to specify which additional packages you need in the image to properly implement health checks (e.g. `jq`?)

Reply:
@dliappis Yes, here it is `jq` as we need to parse the json response from Kibana status. It would be nice if there would be an API from Kibana to get the status which is easier to parse with the default Linux tools, something like `_cat` in ES.